packages/main/src/themes/Tokenizer.css

```diff
@@ -50,7 +50,7 @@
 
 .ui5-tokenizer-more-text {
 	display: inline-block;
-	margin-left: .25rem;
+	margin-inline-start: .25rem;
```

Comment:
margin-inline-start, would this work in IE?

Reply:
Nope, not supported in IE. WIll make separate selectors for RTL.